### PR TITLE
Suggest word use PermissionHelper

### PR DESCRIPTION
**Pull request by ma2tani**

### CODE REVIEW

File: src/main/config/es/fess_config.json [hunk 1]
```diff
@@ -416,7 +416,7 @@
             "type" : "string",
             "index" : "not_analyzed"
           },
-          "targetRole" : {
```
Comment (marevol) [inline src/main/config/es/fess_config.json:419]: targetRole should not be removed because of migration.

**Comment by ma2tani** [comment]
67e7213 "fix use PermissionHelper conflict merge" is [WIP]

File: src/main/java/org/codelibs/fess/app/web/admin/badword/AdminBadwordAction.java [hunk 2]
```diff
@@ -285,6 +285,7 @@ public HtmlResponse upload(final UploadForm form) {
         new Thread(() -> {
             try (Reader reader = new BufferedReader(new InputStreamReader(form.badWordFile.getInputStream(), getCsvEncoding()))) {
                 badWordService.importCsv(reader);
+                suggestHelper.deleteAllBadWords();
```
Comment (marevol) [inline src/main/java/org/codelibs/fess/app/web/admin/badword/AdminBadwordAction.java:288]: See storeAllBadWords().
deleteAllBadWords is invoked in it.

File: src/main/java/org/codelibs/fess/helper/SuggestHelper.java [hunk 3]
```diff
@@ -229,10 +229,7 @@ public void addElevateWord(final String word, final String reading, final String
         }
         final List<String> roleList = new ArrayList<>();
         if (StringUtil.isNotBlank(roles)) {
-            final String[] array = roles.trim().split(",");
-            for (final String role : array) {
-                roleList.add(role);
-            }
+            roleList.add(roles);
```
Comment (marevol) [inline src/main/java/org/codelibs/fess/helper/SuggestHelper.java:232]: Is it correct?
I think roles needs to be splitted by LF.

**Pull request merged**